### PR TITLE
Add spanish translation

### PR DESCRIPTION
**Pull request by mauriciolanda4**
Added the Spanish translation for the app and also added the proper connections to the i18n files

### CODE REVIEW

**Comment by aarrtthhuurr** [comment]
Merci beaucoup pour ta contribution !
La mise à jour de `react-native` est-elle vraiment nécessaire dans `package.json` ?

**Comment by NastuzziSamy** [comment]
Thx again for the contribution ! :)

File: package.json [hunk 1]
```diff
@@ -28,7 +28,7 @@
     "i18n-js": "^3.3.0",
     "react": "16.8.3",
     "react-dom": "^16.8.6",
-    "react-native": "0.59.10",
```
Comment (NastuzziSamy) [inline package.json]: This will break our compilation

**Review by NastuzziSamy — APPROVED** [review]
I need to set react-native as it was
You cannot update a package like that :p

**Review by NastuzziSamy — CHANGES_REQUESTED** [review]
Pls @mauriciolanda4, set `react-native` as it was so we can merge your PR :)

**Comment by mauriciolanda4** [comment]
Will do so, sorry for the late response, my notifications are wonky, and yes it was a mistake that scenario in the package.json, theory is that it auto-updated when I installed it or something.

**Comment by mauriciolanda4** [comment]
Done :D, it seems that the package.json is out-of-date now, and i miss clicked the merging :(

**Comment by mauriciolanda4** [comment]
Is there something I can to do to solve that or is that on your end?

File: package.json (hunk 1 above)
Comment (mauriciolanda4) [reply package.json]: Yes, it is outdated now as well, are you able to modify that file apart from the commit i recently did?, so that it passes the tests

**Comment by mauriciolanda4** [comment]
I'll be attentive to this PR if there is something needed to do

**Comment by mauriciolanda4** [comment]
In case of emergency I could open another PR if needed

**Comment by NastuzziSamy** [comment]
Thx @mauriciolanda4 , checking :)

**Comment by NastuzziSamy** [comment]
Okay, it fails bcs you pushed `package-lock.json`.
I cant update your repository, so can you exceptionnaly add me as contributor in your repo ? 
Moreover, you made 13 commits for one translation, it is really a lot so you/I need to squash this

**Comment by mauriciolanda4** [comment]
Sorry for the Late Reply, my notifications are wonky, sure i will add you as a contributor

**Comment by mauriciolanda4** [comment]
Done, I added you as a collaborator

**Comment by NastuzziSamy** [comment]
Thank you, we are preparing the next release, so i will try to fix your PR as fast as I can. Is it ok if I add as a spanish translator in the app ?

**Comment by mauriciolanda4** [comment]
Yes please, thank you

**Comment by mauriciolanda4** [comment]
If you could show me some screenshots once it is done and live I would appreciate it :D

**Comment by NastuzziSamy** [comment]
Ok I close this PR bcs of all the commits, I am opening a new one and pinging you @mauriciolanda4 :)

**Comment by NastuzziSamy** [comment]
@mauriciolanda4 
70914693_391890361498211_5093848418694987776_n
71106381_536055543803165_8194974196938309632_n

**Comment by mauriciolanda4** [comment]
Thank You

**Comment by mauriciolanda4** [comment]
spanish is missing in the README.md file, that could be edited and added onto any commit

**Pull request closed without merging**